frontend/cypress/integration/featureFiles/app_details.feature

```diff
@@ -37,7 +37,7 @@ Feature: Kiali App Details page
     Then user sees trace details
 
   @app-details-page
-  Scenario: See span info after selecting a span
+  Scenario: See trace info after selecting a span
```

Comment:
```suggestion
  Scenario: See span info after selecting app span
```

Reply:
And possibly a similar change in the conflicting service_details.feature.

Reply:
fixed both files